File: exercise03/printing_quotes.go
package main

import (
	"bufio"
	"fmt"
	"os"
)

func main() {
	scanner := bufio.NewScanner(os.Stdin)
	fmt.Print("What is the quote? ")
	scanner.Scan()
	quote := scanner.Text()

	fmt.Print("Who said it? ")
	scanner.Scan()
	author := scanner.Text()

	result := printQuote(author, quote)
	fmt.Println(result)
}

func printQuote(author, quote string) string {
	return fmt.Sprintf("%v says, \"%v\"", author, quote)
}

File: javascript/exercise03/printing-quotes.js
import { input } from "pynput";

const quote = await input("What is the quote? ");
const author = await input("Who said it? ");

const result = print_quote(author, quote);
console.log(result);

/**
 * @param {string} quote
 * @param {string} author
 * @returns {string}
 */
function print_quote(author, quote) {
	return `${author} says, \"${quote}\"`;
}
